src/include/migraphx/op/argmax.hpp

```diff
@@ -87,6 +89,10 @@ struct argmax
                 max_val   = cur_val;
                 max_index = i;
             }
+            if(select_last_index and float_equal(max_val, cur_val))
```

Comment:
```suggestion
            else if(select_last_index and float_equal(max_val, cur_val))
```

IMO, doesn't make a difference but makes it more clear.